htmresearch/frameworks/nlp/model_factory.py

```diff
@@ -21,13 +21,19 @@
 
 import simplejson
 
-from htmresearch.frameworks.nlp.classify_htm import ClassificationModelHTM
-from htmresearch.frameworks.nlp.classify_keywords import (
-  ClassificationModelKeywords)
+from htmresearch.frameworks.nlp.classification_model import ClassificationModel
 from htmresearch.frameworks.nlp.classify_document_fingerprint import (
   ClassificationModelDocumentFingerprint)
 from htmresearch.frameworks.nlp.classify_fingerprint import (
   ClassificationModelFingerprint)
+from htmresearch.frameworks.nlp.classify_htm import (
+  ClassificationModelHTM)
+from htmresearch.frameworks.nlp.classify_keywords import (
+  ClassificationModelKeywords)
+from htmresearch.frameworks.nlp.classify_network_api import (
```

Comment:
classify_network_api shouldn't be instantiated - it doesn't implement all the required ClassificationModel methods.